src/CheckoutSdk/Marketplace/OnboardingStatus.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace Checkout.Marketplace
+{
+    public enum OnboardingStatus
```

Comment:
1. Please use .NET enum member naming convention (Camel Case)
2. As it is this shouldn't work - `Please add [EnumMember(Value = "value")]` on each entry so it's correctly serialized/desirialized.